lib/css/components/_stacks-badges.less

```diff
@@ -20,10 +20,6 @@
     border-color: @border;
     background-color: @background;
     color: @color;
-
-    .s-tag--dismiss {
```

Comment:
This was not used anywhere in the product, nor in Core. Furthermore, it was bloating the compiled css file, so I deleted it.